apps/challenges/models.py

```diff
@@ -185,18 +178,10 @@ def is_active(self):
         return False
 
 
-signals.post_save.connect(
-    model_field_name(field_name="evaluation_script")(
```

Comment:
Instead of removing `post_save` callbacks from `models.py` you could create the `restart_workers` callback method in `models.py` and import `aws_utils. restart_workers_signal_callback` only in the callback method.